internal/src/main/java/com/onewelcome/internal/util/TestResultFileCreator.kt

```diff
@@ -0,0 +1,35 @@
+package com.onewelcome.internal.util
+
+import android.annotation.SuppressLint
+import java.lang.System.currentTimeMillis
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+class TestResultFileCreator {
+  companion object {
+    @SuppressLint("ConstantLocale")
+    val formattedTime: String = SimpleDateFormat("HH:mm:ss dd-MM-yyyy", Locale.getDefault()).format(Date(currentTimeMillis()))
+    fun getFileContent(appVersionInfo: AppVersionInfo, osVersionInfo: OSVersionInfo, testResultContent: String?): String {
+      return """
+Test run time $formattedTime
+----------------------------
+Android OS
+
+${osVersionInfo.sdkVersion}
+${osVersionInfo.apiLevel}
+${osVersionInfo.codename}
+${osVersionInfo.type}
+----------------------------
+OMI Showcase app
+
+${appVersionInfo.version}
+${appVersionInfo.omiSdkVersion}
+----------------------------
+Test results
+
+$testResultContent
+"""
```

Comment:
indentation is a bit annoying here 😄We could use either StringBuilder which is a bit more efficient or I found this solution:
```
return """
        |Test run time $formattedTime
        |----------------------------
        |Android OS
        |
        |${osVersionInfo.sdkVersion}
        |${osVersionInfo.apiLevel}
        |${osVersionInfo.codename}
        |${osVersionInfo.type}
        |----------------------------
        |OMI Showcase app
        |
        |${appVersionInfo.version}
        |${appVersionInfo.omiSdkVersion}
        |----------------------------
        |Test results
        |
        |$testResultContent
        """.trimMargin()
```

Reply:
Changed into StringBuilder